.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,10 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "Python 3",
+	"image": "mcr.microsoft.com/devcontainers/python:1-3.12-bullseye",
+	"features": {
+		"ghcr.io/guiyomh/features/just:0": {},
+		"ghcr.io/va-h/devcontainers-features/uv:1": {}
```

Comment:
I am not particularly familiar with how devcontainers get composed like this. Do you know if there's a 3.13 container available, for consistency's sake? Or, I could create a dev version of the production Dockerfile we could use as a dev container.

Reply:
I am mostly developing in a Nix shell at the moment but I'm not completely happy with the marriage of Nix and uv that's currently in place.

Reply:
Ah! 3.13 isn't listed in their docs, but it does indeed exist. Updated.

Reply:
I did the Nix thing for a while but it wasn't for me. I'm all in on devcontainers now.